### PR TITLE
build v20.1.0

### PR DESCRIPTION
**Pull request by chrislarrycarl**
This is our "next" update for the build module. It includes updates for all dependencies, most notably adding support for [Babel 7.8](https://github.com/babel/babel/releases/tag/v7.8.0). Additionally linting rules have been split by environment allowing us to clean up a few linting warnings in our base files so that builds are cleaner.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,82 +1,103 @@
...
+    "webpack-dashboard": "^3.2.0",
+    "webpack-dev-server": "^3.10.1",
     "webpack-import-glob": "^2.0.0",
-    "webpack-merge": "4.2.1",
-    "webpack-notifier": "1.7.0"
-  }
+    "webpack-merge": "4.2.2",
+    "webpack-notifier": "^1.8.0"
+  },
+  "browserslist": [
```
Comment (nomad-mystic) [inline package.json:97]: Can't this be removed since it's already set in the Babel config?

Comment (chrislarrycarl) [reply package.json:97]: Good catch @nomad-mystic but it's the inverse. I neglected to delete targets from our babel config when I consolidated to the package.json. This way our JS and CSS will be targeting the same clients with the same settings.

Comment (codyrobert) [reply package.json:97]: Thanks @nomad-mystic @chrislarrycarl - yes, package.json as canonical is preferred, as the browserslist setting is used by postcss as well as babel

**Comment by codyrobert** [comment]
This looks great @chrislarrycarl  - I've been able to compile all of our projects with no issues.  🌮

**Review by codyrobert — APPROVED** [review]

**Pull request merged**